core/src/main/java/nl/nn/adapterframework/validation/AbstractXmlValidator.java

```diff
@@ -59,6 +61,7 @@
 	public static final String XML_VALIDATOR_NOT_VALID_MONITOR_EVENT = "Invalid XML: does not comply to XSD";
 	public static final String XML_VALIDATOR_VALID_MONITOR_EVENT = "valid XML";
 
+	private ClassLoader configurationClassLoader = Thread.currentThread().getContextClassLoader();
```

Comment:
Ik denk dat het safer is om in plaats van Thread.currentThread().getContextClassLoader() de classloader in de constructor mee te geven